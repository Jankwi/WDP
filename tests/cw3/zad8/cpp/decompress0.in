5
1 6 9 42 3