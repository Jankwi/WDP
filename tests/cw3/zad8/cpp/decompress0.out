1 2 2 5 11 11 2